### PR TITLE
webcamoid: init at 8.7.1

### PR DESCRIPTION
**Pull request by robaca**
Expression for packaging webcamoid (https://webcamoid.github.io/)

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by robaca** [comment]
related to package request #84890

**Comment by robaca** [comment]
Is there missing something in the MR to be eligible for a review?

File: pkgs/applications/video/webcamoid/default.nix [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    gstreamer gst_all_1.gst-plugins-base
+    alsaLib libpulseaudio jack2
+    v4l-utils
+  ];
+
+  nativeBuildInputs = [ pkgconfig qmake ];
+
+  qmakeFlags = [ 
+    "Webcamoid.pro" 
+    "INSTALLQMLDIR=$(out)/lib/qt/qml"
```
Comment (aanderse) [inline pkgs/applications/video/webcamoid/default.nix:29]: Please take a look into `${placeholder "out"}` to replace `$(out)`.

File: pkgs/applications/video/webcamoid/default.nix [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  preFixup = ''
+    cat <<EOF > $out/bin/qt.conf
+    [Paths]
+    Imports = ../lib/qt/qml
+    Qml2Imports = ../lib/qt/qml
+    EOF 
+  '';
+
+  enableParallelBuilding = true;
```
Comment (aanderse) [inline pkgs/applications/video/webcamoid/default.nix:40]: Already implied.

**Review by aanderse — CHANGES_REQUESTED** [review]
Just a large backlog of PRs, unfortunately. I haven't looked at the `qt` stuff in a bit so I can't offer more advice, but I'm a bit confused by the `preFixup` and `INSTALLQMLDIR` bit (not saying it is wrong, I'm really just not familiar).

Maybe a wonderful person who is much more knowledgeable than myself and volunteers themselves for PR reviews, like @jonringer, would be willing to take a look.

File: maintainers/maintainer-list.nix [hunk 3]
```diff
@@ -6496,6 +6496,12 @@
     githubId = 353885;
     name = "Rob Vermaas";
   };
+  robaca = {
```
[the remark this answers is not in the document]
Comment (robaca) [reply maintainers/maintainer-list.nix:6499]: Is this just general best practice or do I have to create a new PR? Might have overread this in the docs.

[the remark this answers is not in the document]
Comment (aanderse) [reply maintainers/maintainer-list.nix:6499]: No need for a new PR, just rewrite the commit history.

[the remark this answers is not in the document]
Comment (robaca) [reply maintainers/maintainer-list.nix:6499]: rewritten history

**Comment by aanderse** [comment]
@GrahamcOfBorg build webcamoid

**Comment by aanderse** [comment]
@robaca can you please squash the last 2 commits into a single and have it as `webcamoid: init at 8.7.1` as per [CONTRIBUTING.MD](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

**Comment by robaca** [comment]
@aanderse I fixed the commit history as requested.

**Comment by aanderse** [comment]
@robaca thanks for you patience in all this and the great new package. Welcome to the team! :tada:

**Pull request merged**